opensearch_py_ml/ml_commons/rag_pipeline/rag/ingest.py

```diff
@@ -0,0 +1,194 @@
+# ingest_class.py
```

Comment:
Add license header

Reply:
Completed, Added license header to all files.